src/ch05-02-example-structs.md

```diff
@@ -225,42 +242,44 @@ struct Rectangle {
 fn main() {
     let rect1 = Rectangle { width: 30, height: 50 };
 
-    println!("rect1 is {:?}", rect1);
+    println!("rect1 to {}", rect1);
 }
 ```
 
-<span class="caption">Listing 5-12: Adding the annotation to derive the `Debug`
-trait and printing the `Rectangle` instance using debug formatting</span>
+<span class="caption">Listing 5-12: Dodanie adnotacji derywacji cechy `Debug`
```

Comment:
nie znałem słowa derywacja, ale wg. słownika j. pol. ma ono niepasujące nam znaczenia https://sjp.pwn.pl/szukaj/derywacja.html
 Może: Dodanie adnotacji nadającą cechę (warto to dodać do listy tłumaczeń zwrotów i ewentualnie tam dyskutować)

Reply:
też szukałem i natrafiłem na ten sam wniosek, ale pozbywamy się tutaj terminologii, koniec końców ktoś będzie szukał błędów na stack overflow i o wiele łatwiej będzie mu połączyć terminologię `derywacji` i adnotacje `derive` kiedy je spotka, ale mimo wszystko twoja sugestia brzmi zdecydowanie lepiej

Reply:
zmienione